pkg/cli/testdata/MockClient.go

```diff
@@ -400,58 +472,17 @@ func (m *MockClient) ImageList(ctx context.Context) ([]apiv1.Image, error) {
 }
 
 func (m *MockClient) ImageGet(ctx context.Context, name string) (*apiv1.Image, error) {
-	switch name {
-	case "dne":
-		return nil, nil
-	case "found.image":
-		return &apiv1.Image{}, nil
-	case "found.image-two-tags":
-		return &apiv1.Image{}, nil
-	case "testtag":
-		return &apiv1.Image{
-			TypeMeta:   metav1.TypeMeta{},
-			ObjectMeta: metav1.ObjectMeta{Name: "found-image1234567"},
-			Tags:       []string{"testtag:latest"},
-			Digest:     "1234567890asdfghkl",
-		}, nil
-	case "testtag1":
-		return &apiv1.Image{
-			TypeMeta:   metav1.TypeMeta{},
-			ObjectMeta: metav1.ObjectMeta{Name: "found-image-two-tags1234567"},
-			Tags:       []string{"testtag1:latest", "testtag2:v1"},
-			Digest:     "lkjhgfdsa1234567890",
-		}, nil
-	case "lkjhgfdsa1234567890":
-		return &apiv1.Image{
-			TypeMeta:   metav1.TypeMeta{},
-			ObjectMeta: metav1.ObjectMeta{Name: "found-image-two-tags1234567"},
-			Tags:       []string{"testtag1:latest", "testtag2:v1"},
-			Digest:     "lkjhgfdsa1234567890",
-		}, nil
-	case "index.docker.io/subdir/test:v1":
-		return &apiv1.Image{
-			TypeMeta:   metav1.TypeMeta{},
-			ObjectMeta: metav1.ObjectMeta{Name: "registy1234567-two-tags"},
-			Tags:       []string{"index.docker.io/subdir/test:v1", "index.docker.io/subdir/test:v2"},
-			Digest:     "registry1234567",
-		}, nil
-	case "registry1234567":
-		return &apiv1.Image{
-			TypeMeta:   metav1.TypeMeta{},
-			ObjectMeta: metav1.ObjectMeta{Name: "registy1234567-two-tags"},
-			Tags:       []string{"index.docker.io/subdir/test:v1", "index.docker.io/subdir/test:v2"},
-			Digest:     "registry1234567",
-		}, nil
+	if m.ImageItem != nil {
+		return m.ImageItem, nil
 	}
 	return nil, nil
 }
 
 func (m *MockClient) ImageDelete(ctx context.Context, name string, opts *client.ImageDeleteOptions) (*apiv1.Image, error) {
+	if m.ImageItem != nil {
+		return m.ImageItem, nil
+	}
 	switch name {
-	case "dne":
-		return nil, nil
-	case "found-image1234567":
-		return &apiv1.Image{}, nil
 	case "found-image-two-tags1234567":
```

Comment:
Maybe we could export `const` values for the common switch values that these can use? That would make the package a bit more discoverable for new devs coming in to write tests. That way as a developer I could name my object `testdata.FoundWithTwoTags` or something akin to that. Thoughts?